lib/jormungandr/test/integration/Test/Integration/Jormungandr/Scenario/CLI/Server.hs

```diff
@@ -45,21 +45,49 @@ import Test.Integration.Framework.DSL
     , KnownCommand (..)
     , cardanoWalletCLI
     , collectStreams
+    , eventually
     , expectPathEventuallyExist
     , proc'
     , shouldContainT
     , shouldNotContainT
     )
 import Test.Integration.Framework.TestData
-    ( versionLine )
+    ( cmdOk, versionLine )
 import Test.Utils.Ports
     ( findPort )
 
 import qualified Data.Text as T
+import qualified Data.Text.IO as TIO
 
 spec :: forall t. KnownCommand t => SpecWith (Context t)
 spec = do
     block0H <- runIO $ T.unpack <$> getBlock0HText
+    let configForLaunch = "test/data/jormungandr/config-for-passive-launch.yaml"
```

Comment:
We should use OS-agnostic path separator here to make the reference works fine on all platforms!

```hs
import System.FilePath((</>))

let configForLaunch = "test" </> "data" </> ...
```